swarm/storage/ldbstore_test.go

```diff
@@ -491,9 +491,10 @@ func TestLDBStoreRemoveThenCollectGarbage(t *testing.T) {
 	cleanup()
 
 	ldb, cleanup = newLDBStore(t)
+	capacity = 10
 	ldb.setCapacity(uint64(capacity))
 
-	n = 10
+	n = 11
```

Comment:
I vaguely remember that numbers here didn't match up back when we were adding those tests, and now I know the `entryCnt` is the reason for that. In this test, we are setting capacity to 10 for LDBStore, adding 11 elements, and make sure that we have garbage collected one element.